<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="levelLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>level</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="Lawn" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>1000</width>
      <height>430</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sunpointsLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="p1Button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p2Button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p7Button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p6Button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p5Button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p3Button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p4Button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="p8Button">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>20</y>
      <width>130</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="userComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Lawn</class>
   <extends>QGraphicsView</extends>
   <header>lawn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
